package.json

```diff
@@ -61,12 +62,14 @@
     "jest-image-snapshot": "^6.1.0",
     "lerna": "^8.0.2",
     "lint-staged": "^10.5.4",
+    "node-gyp": "^10.0.0",
```

Comment:
Is this still needed?

Reply:
Hmmm, works without it, but please mind, it's gonna stay in our deps anyway. I can see it was added in #2478.